src/components/inputs/toggle-input/toggle-input.js

```diff
@@ -1,8 +1,100 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { css } from '@emotion/core';
-import ToggleSwitch from './toggle-switch';
+import styled from '@emotion/styled';
 import filterDataAttributes from '../../../utils/filter-data-attributes';
+import vars from '../../../../materials/custom-properties';
+
+const thumbSmallSize = '13px';
+const thumbBigSize = '26px;';
+
+const Label = styled.label`
+  position: relative;
+  display: inline-block;
+  cursor: ${props => (props.isDisabled ? 'not-allowed' : 'pointer')};
+
+  ${props =>
+    props.size === 'small'
+      ? `
+        height: 16px;
+        width: 32px;
+      `
+      : `
+
+        height: 32px;
+        width: 64px;
```

Comment:
Can we maybe "correlate" those sizes? Like in L7 have `thumbSmallSize = 13. Then `thumbBigSize = thumbSmallSize * 2` and then move this up as `labelSize = 16` while multiplying width and height again while everywhere appending the `px`. So that it's clearer how sizes relate.

Reply:
Yeah, that's a good improvement. Thanks 😄